src/backend/renderer/gles2/mod.rs

```diff
@@ -1627,6 +1632,108 @@ impl Offscreen<Gles2Renderbuffer> for Gles2Renderer {
     }
 }
 
+impl<B1, B2> Blit<B1, B2> for Gles2Renderer
+where
+    Self: Bind<B1> + Bind<B2>,
+{
+    fn blit(
+        &mut self,
+        from: B1,
+        to: B2,
+        src: Rectangle<i32, Physical>,
+        dst: Rectangle<i32, Physical>,
+        filter: TextureFilter,
+    ) -> Result<(), <Self as Renderer>::Error> {
+        // glBlitFramebuffer is sadly only available for GLES 3.0 and higher
```

Comment:
Would be nice to state the features gated by GL versions and features.

Reply:
I have quite a few renderer-specific doc comments on some trait-implementations for the `Gles2Renderer`, sadly rustdoc doesn't render any of them: https://smithay.github.io/smithay/smithay/backend/renderer/gles2/struct.Gles2Renderer.html#trait-implementations ...